### DISCUSSION TITLE
Removal of `uiForEach`

### DISCUSSION THREAD

**Discussion by simplexidev** [libui-ng/libui-ng#138] [Ideas]
I think it would be just as efficient to use a boolean value instead of defining another enumeration where:

`uiForEachContinue` is replaced with 0,
`uiForEachStop` is replaced with 1.

I use `int` values because there are not actual `bool` usage in the header files. Also, values could be vice-versa if it would be preferred. 

I also don't know the rationale behind the creation of the enumeration anyways, just a thought!

**Comment by rubyFeedback** [comment]
> I also don't know the rationale behind the creation of the enumeration anyways, just a thought!

I don't know either (one problem with lack of comments and documentation) but it may be that
it was more of an ad-hoc solution that may have remained. At the least I have this impression a
bit with regard to some of libui internals, such as the requirement to first build up the menu
interface, before other calls for instance.
